server/src/services/mail/index.js

```diff
@@ -0,0 +1,32 @@
+import nodemailer from "nodemailer";
+import { config } from "dotenv";
+config();
+
+const sendEmail = () =>{
+
+    let transporter = nodemailer.createTransport({
+        host: "smtp.mailtrap.io",
+        port: 2525,
+        auth: {
+          user: process.env.MAIL_USER,
+          pass: process.env.MAIL_USER_PASSWORD
+        }
+    });
+  
+    let mailOptions = {
+      from: '"Plant Lovers" <plantlovers@plantme.com>',
+      to: "daisy@example.com",
+      text: "Can you adopt me?",
+      subject: "Hello Plant Lover 🌼",
+    };
```

Comment:
Beside that in future we can create some email templates for specific email cases, pug library can be helphul for that https://www.npmjs.com/package/pug